eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/map/MutableMapFactory.java

```diff
@@ -72,6 +72,9 @@
 
     <K, V> MutableMap<K, V> ofMap(Map<? extends K, ? extends V> map);
 
+    /**
+     * @since 10.3.0
+     */
```

Comment:
Not accurate. @nikhilnanivadekar added this in 2018.

Reply:
My bad! shouldn't be here! fixing it ...